frame/custom-signatures/src/ethereum.rs

```diff
@@ -0,0 +1,72 @@
+//! Ethereum prefixed signatures compatibility instances.
+
+use codec::{Decode, Encode};
+use sp_core::ecdsa;
+use sp_io::{crypto::secp256k1_ecdsa_recover_compressed, hashing::keccak_256};
+use sp_runtime::traits::{IdentifyAccount, Lazy, Verify};
+use sp_runtime::MultiSignature;
+
+/// Ethereum-compatible signature type.
+#[derive(Encode, Decode, PartialEq, Eq, Clone)]
+pub struct EthereumSignature(pub [u8; 65]);
```

Comment:
Could you resolve this line build error?

Reply:
Hmm, it compiled well on my side, probably rust version is different